components/shared/SquareButton.js

```diff
@@ -63,22 +63,32 @@ const SquareButton = ({
 
 SquareButton.propTypes = {
   className: PropTypes.string,
-  backgroundColor: PropTypes.string.isRequired,
```

Comment:
Thanks for getting this fixed! Trying to get this filled in when I touch a component, sometimes copy/paste fails you 😂